src/Upbit/Spot/Spot.jl

```diff
@@ -0,0 +1,23 @@
+module Spot
+
+using CryptoAPIs.Upbit
+
+"""
+    public_client = UpbitClient(; base_url = "https://api.UPbit.com")
+"""
+const public_client =
+    UpbitClient(; base_url = "https://api.UPbit.com")
```

Comment:
Can you, please, share API reference for this URL? I'm just curious where it comes from

Reply:
URLs are supposed to be case-sensitive. It would be unfortunate to have issues because of that

Reply:
```suggestion
    UpbitClient(; base_url = "https://api.upbit.com")
```

Reply:
@dmitrii-doronin I found some examples of using API in [Upbit](https://docs.upbit.com/docs/upbit-quotation-restful-api) and it is really used there `https://api.upbit.com`